### PR TITLE
[Integration test] Update integration test server and add attendee presence timeout to audio, video test only

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**
- Currently, some canary runs are failing with the error:
```
Unable to create meeting, server error: Error: Request failed with status code 403
```

- This is due to the incorrect handling of baseURL in the integration server.

The current demo URL is (incorrect URL hence error from server):
```
<server>/?attendee-presence-timeout-ms=5000join?title=<title>&name=<name>&region=<region>
```
After this fix it will be:
```
<server>/join?attendee-presence-timeout-ms=5000&title=<title>&name=<name>&region=<region>
```

**Description of changes:**
- This fix handles the query params if any and accordingly generates the create meeting URL.
- Adds attendee presence timeout to the demo url when running Audio and Video test only. The canary demo link is now changed back to just the URL with no query params.
- The `integration/js/server.js` change can be reverted because we reverted the query parameter on the gamma demo URL, but it will catch such cases where we add any other query parameter to demo URLs and run test against it.

**Testing**
1. Have you successfully run `npm run build:release` locally? Yes.
2. How did you test these changes?
- By running the integration tests on the local selenium grid.

3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? NA. This is integration test change only.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? NA.
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? NA.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: integration/js/server.js [hunk 1]
```diff
@@ -33,13 +33,19 @@ createMeeting = async (baseUrl) => {
   }
   meetingCreationStatus = "creating";
   meetingTitle = uuidv4();
-  const createMeetingUrl = `${baseUrl}join?title=${meetingTitle}&name=MeetingOwner&region=us-east-1`;
+  let url = `${baseUrl}`;
+  let index = url.indexOf('?');
```
Comment (michhyun1) [inline integration/js/server.js:37]: url.includes('?')

Comment (michhyun1) [reply integration/js/server.js:37]: Can we use this instead?

Comment (devalevenkatesh) [reply integration/js/server.js:37]: I needed the index for the `substring` so used `indexOf`.

File: integration/js/utils/SdkBaseTest.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ class SdkBaseTest extends KiteBaseTest {
       this.userArn = this.payload.userArn;
     }
 
+    if (['Video', 'Audio'].includes(testName)) {
```
Comment (michhyun1) [inline integration/js/utils/SdkBaseTest.js:23]: Just double checking - this is not backwards? Will 'testname' be Video or Audio?

Comment (michhyun1) [reply integration/js/utils/SdkBaseTest.js:23]: Just checked - our Audio and Video E2E test use the name "Audio" and "Video". Thanks

**Review by michhyun1 — APPROVED** [review]

**Pull request merged**